docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+---
+id: googledrive
+title: Google Drive connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive connector** allows you to create empty folders or documents from templates from your BPMN process.
+
+## Prerequisites
+
+To start working with the **Google Drive connector**, a relevant OAuth token must be configured and stored as a secret in the user's cluster. The token has to have permission to read and create a folder and/or files from a desired Google Drive instance. Follow the steps from the appendix to find out more about creating an OAuth token and giving relevant permissions.
+
+## Create a Google Drive connector task
+
+Currently, the **Google Drive connector** supports two flavors: create a new folder and create a file from a template.
```

Comment:
What do you mean by "flavors" here? Use cases? Instances?

Reply:
Two variants of templates (and use cases) - one to create a folder, one to create a file.

Reply:
Excellent! I would phrase it like this in the doc then.

"Currently, the **Google Drive connector** supports two types of templates: create a folder and create a file from a template."

Reply:
Updated